Add awk PIE build (#102)
unikraft/dynamic-apps: awk/README.md

> # Awk as Dynamic (PIE) Executables

Reviewer: ```suggestion
# AWK as Dynamic (PIE) Executables
```

> This directory stores the binary executable (ELF - Executable and Linkable Format), dynamic library dependencies and support files to run the applications provided by iputils.

Reviewer: ```suggestion
This directory stores the binary executable (ELF - Executable and Linkable Format), dynamic library dependencies and support files to run AWK.
```

The binary is PIE (Position-Independent Executable).
All files are stored in their expected location from the Linux filesystem hierarchy.
The aim is to run the executable in any Linux ELF-compatible environment: native Linux, chroot/jail, Docker, Unikraft.

> ## Run a sample awk command Natively on Linux

Reviewer: ```suggestion
## Run a Sample AWK command Natively on Linux
```

> Run the binary using (for example) the following commands:

Reviewer: ```suggestion
Run the binary using (for example) the following command:
```

> Run the binaries using (for example) the following commands:

Reviewer: ```suggestion
Run the binary using (for example) the following commands:
```

```console

> awk 'BEGIN { print "Hello World" }'

Reviewer: ```suggestion
./usr/bin/awk 'BEGIN { print "Hello World" }'
./usr/bin/awk -f ./hello_world.awk
```

```

> ## Run awk with Unikraft

Reviewer: ```suggestion
## Run AWK with Unikraft
```

> To run the binary ELF on Unikraft, run the `run.sh` script from the [`run-app-elfloader` repository](https://github.com/unikraft/run-app-elfloader).

Reviewer: ```suggestion
To run the AWK binary ELF on Unikraft, use the `run.sh` script from the [`run-app-elfloader` repository](https://github.com/unikraft/run-app-elfloader).
```
